apps/www/registry/tui/ui/input.tsx

```diff
@@ -156,11 +156,11 @@ const Input = React.forwardRef<
               <span className={cn("inline-flex items-center  rounded-l-md border border-r-0 px-3 sm:text-sm", className)}{...props}>
                 {addOnLabel}</span> : null
           }
-          {icon ? <Icon name={icon} className={cn(`${alignIcon === "left" ? "absolute top-[37px] left-0 flex items-center pl-3" : "mr-2 absolute top-[37px] right-0 flex items-center pl-3"} ${iconStyle}`, fontColor(color), className)} {...props} /> : null}
+          {icon ? <Icon name={icon} className={cn(`${alignIcon === "left" ? "absolute top-[11px] left-0 flex items-center pl-3" : "mr-2 absolute top-[11px] right-0 flex items-center pl-3"} ${iconStyle}`, fontColor(color), className)} {...props} /> : null}
           {
             alignDropdown && (
               <div className={cn(`absolute text-sm inset-y-0 ${alignDropdown === 'prefix' ? 'left-0 flex items-center' : 'right-0 flex pr-px items-center -mb-[7px]'}`, className)} {...props}>
-                <select className={cn(`mt-${alignDropdown === 'prefix' ? '8' : '6'} ${alignDropdown === 'prefix' ? 'pl-2' : 'pr-2'} bg-inherit`, fontColor(color), className)}>
+                <select className={cn(`mt-${alignDropdown === 'prefix' ? '8' : '6'} ${alignDropdown === 'prefix' ? 'pl-2 -top-[23px]' : 'pr-2 -left-[61px] -top-[14px]'} bg-inherit absolute`, fontColor(color), className)}>
```

Comment:
Don't use constant values in top and left. This will not work when input size is small